public/js/controllers/game.js

```diff
@@ -1,11 +1,13 @@
+/* eslint-disable */
```

Comment:
You have certain violations I believe you could have resolved without having to disable this entire file. You miss out on the learning you should have by virtue of resolving those violations